packages/npm/@amazeelabs/gatsby-source-silverback/directives.graphql

```diff
@@ -2,3 +2,8 @@ directive @sourceFrom(fn: String) on OBJECT
 directive @resolveBy(fn: String) repeatable on FIELD_DEFINITION
 directive @gatsbyNodes(type: String) on FIELD_DEFINITION
 directive @gatsbyNode(type: String, id: String) on FIELD_DEFINITION
+directive @responsiveImageSharp(
```

Comment:
See above. The separate directive should not be necessary. We just provide a different implementation for `@responsiveImage`.

Reply:
Done that now. I think we will probably have to rename the 'transform' parameter to something else, so that we could also use it as an 'options' parameter for other services. Right now we do not use it for sharp, but we might in the future.